cli/ts/commands/proveOnChain.ts

```diff
@@ -144,13 +137,10 @@ export const proveOnChain = async ({
   const numSignUpsAndMessages = await pollContract.numSignUpsAndMessages();
   const numSignUps = Number(numSignUpsAndMessages[0]);
   const numMessages = Number(numSignUpsAndMessages[1]);
-  const messageBatchSize = MESSAGE_TREE_ARITY ** Number(treeDepths.messageTreeSubDepth);
+  const messageBatchSize = MESSAGE_BATCH_SIZE;
```

Comment:
this should not come as a constant but it should be fetched from the `Poll` contract

Reply:
this is not resolved